### PR TITLE
Fixed steps for MongoDB example

### PR DESCRIPTION
**Pull request by shahpratikr**
## Change Overview

- replaced mongo command is replaced by mongosh command due to change in latest mongodb chart
- fixed namespace name in commands

## Pull request type

- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [x] :world_map: Documentation
- [ ] :robot: Test

## Test Plan

- [x] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: examples/mongodb/README.md [hunk 1]
```diff
@@ -121,65 +132,81 @@ To restore the missing data, you should use the backup that you created before.
...
+]
 ```
 
 ### Delete the Artifacts
 
 The artifacts created by the backup action can be cleaned up using the following command:
 
 ```bash
-$ kanctl --namespace kasten-io create actionset --action delete --from "backup-llfb8"
+$ kanctl --namespace kanister create actionset --action delete --from "backup-llfb8" --namespacetargets kanister
```
Comment (viveksinghggits) [inline examples/mongodb/README.md:166]:
```suggestion
$ kanctl create actionset --action delete --from "backup-llfb8" --namespacetargets kanister
```

**Comment by viveksinghggits** [comment]
@shahpratikr do we need changes in the blueprint as well? Also, it looks like you have put PR desc twice.

**Comment by shahpratikr** [comment]
> do we need changes in the blueprint as well?
No changes were needed in blueprint

File: examples/mongodb/README.md (hunk 1 above)
Comment (viveksinghggits) [reply examples/mongodb/README.md:166]: LGTM apart from above comment.

Comment (pavannd1) [reply examples/mongodb/README.md:166]: Don't we need `--namespace` flag for the ActionSet to be created in the `kanister` namespace? `--namespacetargets` only sets it as a subject right?

Comment (shahpratikr) [reply examples/mongodb/README.md:166]: namespace is required here. otherwise it will create actionset in default namespace

Comment (viveksinghggits) [reply examples/mongodb/README.md:166]: ok, makes sense. Thank you for explaining.

**Review by viveksinghggits — APPROVED** [review]

**Pull request merged**